README.md

```diff
@@ -4,62 +4,136 @@
-
-Everything runs locally  with no server support and accelerated with local GPUs on your phone and laptops.
-[Supported platforms](https://github.com/mlc-ai/mlc-llm/issues/15) include:
-* iPhone, iPad;
-* Android phones;
-* Apple Silicon and x86 MacBooks;
-* AMD, Intel and NVIDIA GPUs via Vulkan on Windows and Linux;
-* NVIDIA GPUs via CUDA on Windows and Linux;
-* WebGPU on browsers (through companion project [WebLLM](https://github.com/mlc-ai/web-llm/tree/main)).
-
-<ins>**[Check out our instruction page to try out!](https://mlc.ai/mlc-llm/docs/get_started/try_out.html)**</ins>
-
-<p align="center">
-  <img src="site/gif/ios-demo.gif" height="700">
-</p>
+MLC LLM is a high-performance **universal deployment** solution for large-language models that allows **any language models** to be **deployed natively** on a diverse set of hardware backends and native applications, plus a **productive framework** for everyone to further optimize model performance for their own use cases.
+
+The cornerstone of MLC LLM is machine learning compilation (MLC) and the mission of MLC LLM is to **enable everyone to develop, optimize and deploy AI models natively on everyone's devices**.
+
+MLC LLM supports the following platforms and hardware:
+
+<table style="width:100%">
+  <thead>
+    <tr>
+      <th style="width:16%"> </th>
+      <th style="width:21%">AMD GPU</th>
+      <th style="width:21%">NVIDIA GPU</th>
+      <th style="width:21%">Apple M1/M2 GPU</th>
+      <th style="width:21%">Intel GPU</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Linux / Win</td>
+      <td>✅ Vulkan, ROCm</td>
+      <td>✅ Vulkan, CUDA</td>
+      <td>N/A</td>
+      <td>✅ Vulkan</td>
+    </tr>
+    <tr>
+      <td>macOS</td>
+      <td>✅ Metal</td>
+      <td>N/A</td>
+      <td>✅ Metal</td>
+      <td>✅ Metal</td>
+    </tr>
+    <tr>
+      <td>Web Browser</td>
+      <td>✅ WebGPU</td>
+      <td>✅ WebGPU</td>
+      <td>✅ WebGPU</td>
+      <td>✅ WebGPU</td>
+    </tr>
+    <tr>
+      <td>iOS / iPadOS</td>
+      <td colspan=4>✅ Metal on Apple M1/M2 GPU</td>
+    </tr>
+    <tr>
+      <td>Android</td>
+      <td colspan=2>✅ OpenCL on Adreno GPU</td>
+      <td colspan=2>✅ OpenCL on Mali GPU</td>
+    </tr>
+  </tbody>
+</table>
+
+MLC LLM provides multiple sets of APIs across platforms and environments, which you can use to build
+your own LLM applications with native acceleration by MLC LLM. These include:
+* [Python API and gradio frontend](https://mlc.ai/mlc-llm/docs/deploy/python.html),
+* [Rest API with OpenAI API compatibility](https://mlc.ai/mlc-llm/docs/deploy/rest.html),
+* [C++ API](https://mlc.ai/mlc-llm/docs/deploy/cli.html),
+* [JavaScript API](https://mlc.ai/mlc-llm/docs/deploy/javascript.html) and [Web LLM](https://github.com/mlc-ai/web-llm),
+* [Swift API for iOS](https://mlc.ai/mlc-llm/docs/deploy/ios.html), and
+* [Android App](https://mlc.ai/mlc-llm/docs/deploy/android.html).
```

Comment:
```suggestion

## Universal Deployment and APIs

* [Python API](https://mlc.ai/mlc-llm/docs/deploy/python.html);
* [Rest API with OpenAI API compatibility](https://mlc.ai/mlc-llm/docs/deploy/rest.html);
* [C++ API](https://mlc.ai/mlc-llm/docs/deploy/cli.html);
* [JavaScript API](https://mlc.ai/mlc-llm/docs/deploy/javascript.html) and [WebLLM](https://github.com/mlc-ai/web-llm);
* [Swift API and iOS App](https://mlc.ai/mlc-llm/docs/deploy/ios.html);
* [Java API and Android App](https://mlc.ai/mlc-llm/docs/deploy/android.html).
```

Reply:
I’m wondering if it is good to explicit mention that people can do secondary development with the API.

Reply:
I do want to emphasize that our API surface is awesome awesome, but to be clear, those documents haven’t covered APIs yet, unfortunately. Let’s avoid overselling for now